README.md

```diff
@@ -257,13 +180,13 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank Cindy Shin on Behance
```

Comment:
Great job @Zeph180 🤝

- [ ] [Optional] Could you include these clickable links to the author to enable visitors to your project to navigate to the credentials of this author? i.e, the "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] [Optional] You may also fix the broken MIT license link to make your readme file more professional.